### PR TITLE
(+semver: feature) Add support for Omnikey contactless CardReader

### PR DESCRIPTION
**Pull request by annymsMthd**
Signed-off-by: Joshua Benjamin <benjamin@syncromatics.com>

### CODE REVIEW

File: src/Cid/Cid.nuspec [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+   <metadata>
+       <id>Cid</id>
+       <version>$version$</version>
+       <title>Xamarin Android USB peripherals</title>
+       <authors>Syncromatics Development</authors>
```
Comment (thzinc) [inline src/Cid/Cid.nuspec]: Let's use Syncromatics Engineering to stay in line with the "This software is made available by Syncromatics Engineering under the MIT license." line from README.md

File: src/Cid/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Cid")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Syncromatics")]
+[assembly: AssemblyProduct("Cid")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
```
Comment (thzinc) [inline src/Cid/Properties/AssemblyInfo.cs]: Is this auto-generated? If not, make this `Copyright (c) 2017 Syncromatics` to match the LICENSE file

File: example/Cid.Example/Properties/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="Cide.Example.Cide.Example" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
```
Comment (thzinc) [inline example/Cid.Example/Properties/AndroidManifest.xml]: Is `Cide` intentional here or is it a typo?

File: example/Cid.Example/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Android.App;
+
+[assembly: AssemblyTitle("Cide.Example")]
```
Comment (thzinc) [inline example/Cid.Example/Properties/AssemblyInfo.cs]: Is `Cide` intentional here or is it a typo?

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+services:
+  - docker
+
+branches:
+  only: 
+    - master
+
+install:
+  - docker pull syncromatics/build-box:feature_add-xamarin-android-support
```
Comment (thzinc) [inline .travis.yml]: What will the "proper" tag be for this image? `syncromatics/build-box:release-5.4`?

**Review by thzinc — CHANGES_REQUESTED** [review]
Overall, looks like a useful library of tooling. Mostly minor comments and questions here.

**Review by derrickcrowne — DISMISSED** [review]

**Review by derrickcrowne — APPROVED** [review]

File: src/Cid/CardReaders/CardReader.cs [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+            var manager = (UsbManager)Application.Context.GetSystemService(Context.UsbService);
+            _connection = manager.OpenDevice(_device);
+            var claimed = _connection.ClaimInterface(_interface, true);
+            if (!claimed)
+                throw new Exception("Interface could not be claimed");
+
+            var rawDescriptors = _connection.GetRawDescriptors();
+            Descriptor = new CcidDescriptor(rawDescriptors);
+
+            if (!Descriptor.Features.AutoParamConfigViaAtr)
```
Comment (kennethito) [inline src/Cid/CardReaders/CardReader.cs]: Deadcode?

**Review by kennethito — APPROVED** [review]

**Review by thzinc — APPROVED** [review]

**Pull request merged**